support/eligibility-policy.html

```diff
@@ -80,7 +80,11 @@ <h1 class="block__heading__title">Support Denial</h1>
 			</tr>
 			<tr>
 				<td>Danger to privacy/safety/security</td>
-				<td>Content on forum may pose a danger to privacy, safety, or security of others</td>
+				<td>Forum contains content that may pose a danger to privacy, safety, or security of others</td>
+			</tr>
+			<tr>
+				<td>Children abuse related material</td>
+				<td>Forum contains children abuse or children sexual abuse material, as well as children content in a sexualized manner</td>
```

Comment:
Usually in this context the singular "child" is used. Suggest changing these two lines to:

```html
				<td>Child abuse or sexualisation</td>
				<td>Forum contains child abuse or child sexual abuse content, and/or content presenting children in a sexualised manner or context</td>
```